pkg/volume/azure_dd/attacher.go

```diff
@@ -224,7 +224,7 @@ func (attacher *azureDiskAttacher) MountDevice(spec *volume.Spec, devicePath str
 	if notMnt {
 		diskMounter := &mount.SafeFormatAndMount{Interface: mounter, Runner: exec.New()}
 		mountOptions := volume.MountOptionFromSpec(spec, options...)
-		err = diskMounter.FormatAndMount(devicePath, deviceMountPath, *volumeSource.FSType, mountOptions)
+		err = diskMounter.FormatAndMount(devicePath, deviceMountPath, *volumeSource.FSType, mountOptions, []string{"-E", "lazy_itable_init=0,lazy_journal_init=0", "-F"})
```

Comment:
For reference, I formatted a 100GB Premium SSD on an F instance, repeated 3 times and time was consistent:
```console
# time mkfs.ext4 -F -E lazy_itable_init=0,lazy_journal_init=0 /dev/sdf 2>&1 > /dev/null
mke2fs 1.42.9 (28-Dec-2013)

real    0m18.770s
user    0m0.172s
sys     0m1.207s

# time mkfs.ext4 -F  /dev/sdf 2>&1 > /dev/null
mke2fs 1.42.9 (28-Dec-2013)

real    0m1.840s
user    0m0.158s
sys     0m0.118s
```